payu_payment_methods/lib/src/features/payment_methods/payment_methods_controller.dart

```diff
@@ -44,7 +44,6 @@ class PaymentMethodsController extends PayuController {
       PaymentMethodValue.applePay,
       PaymentMethodValue.blikCode,
       PaymentMethodValue.googlePay,
-      PaymentMethodValue.mastercardInstallments,
```

Comment:
To zostaw ponieważ `PaymentMethodValue.mastercardInstallments` jest źle nazwany. 
```
abstract class PaymentMethodValue {
  static const applePay = 'jp';
  static const blikCode = 'blik_code';
  static const googlePay = 'ap';
  static const mastercardInstallments = 'ai';
}
```
`PaymentMethodValue.mastercardInstallments` powinno się nazywać `PaymentMethodValue.installments` bo ma wartość `ai` czyli zwykłe raty a został nieprawidłowo użyty do rat Mastercard